package.json

```diff
@@ -71,44 +70,44 @@
 		"publish": "vsce publish patch"
 	},
 	"devDependencies": {
```

Comment:
@Will-Sommers you're going to have a bunch of merge conflicts with #28; not sure how to handle.  I'd be inclined to get this PR in because it's fairly mechanical, but you're probably then going to need to redo the pins on #28.